DashAI/back/dependencies/registry/relationship_manager.py

```diff
@@ -66,6 +70,43 @@ def add_relationship(
         self._relations[first_component_id].append(second_component_id)
         self._relations[second_component_id].append(first_component_id)
 
+    @beartype
+    def remove_relationship(
+        self, first_component_id: str, second_component_id: str
+    ) -> None:
+        """Remove an existing relation to the relationship manager.
+
+        Parameters
+        ----------
+        first_component_id : str
+            First component id or name.
+        second_component_id : str
+            Second component id or name.
+
+        """
+        try:
+            self._relations[first_component_id].remove(second_component_id)
+        except KeyError as e:
+            logger.error(
+                f"Error: Relationship between {first_component_id} and does "
+                f"not exist {second_component_id} in the registry. Exception: "
+                f"{e}"
+            )
+
+        try:
+            self._relations[second_component_id].remove(first_component_id)
+        except KeyError as e:
+            logger.error(
+                f"Error: Relationship between {second_component_id} and does "
+                f"not exist {first_component_id} in the registry. Exception: "
+                f"{e}"
+            )
```

Comment:
Same as before: here you should use ValueError to report the error. 
Just to comment: these operations in a normal operation should always work, so if this case occurs, it implies that something is wrong and therefore, it should be notified by means of an exception.